### ISSUE TITLE
Handle jumps into try catch blocks

### ISSUE THREAD

**Issue by GraxCode** [leibnitz27/cfr#202]
# Problem solved by this feature
Jumps into the middle of try catch blocks like this could be reproduced:
```java
    public static void main(String[] args) {
        System.out.println(0);
        if (args != null) ** GOTO lbl6
        System.lineSeparator();
        ** GOTO lbl8
lbl6:
        // 1 sources

        try {
            System.exit(0);
lbl8:
            // 2 sources

            System.exit(1);
        }
        catch (Throwable t) {
            t.printStackTrace();
        }
    }
```

# Feature description
https://github.com/leibnitz27/cfr/blob/master/src/org/benf/cfr/reader/bytecode/analysis/opgraph/op3rewriters/JumpsIntoDoRewriter.java can also be implemented for try catch blocks. 

# Considerations
This is possible, because `if(boolean)` won't throw any exception. On small code (like in this example), code duplication would be a better solution, as it produces cleaner code.
[jump-in-tcb.jar.txt](https://github.com/leibnitz27/cfr/files/5011857/jump-in-tcb.jar.txt)

**Comment by leibnitz27** [comment]
Of course, the annoying thing here is that System.lineSeparator can be shown with trivial static analysis to be no-throw, so everything after the first println COULD be moved into the try block as-is.

(Of course, that's only the case for this specific example - In practice I would avoid doing that anyway - I have avoided external static analysis because it relies on methods not being replaced at runtime/multijar concerns/sheer possible cost).
